irohad/main/application.cpp

```diff
@@ -184,7 +184,8 @@ void Irohad::initOrderingGate() {
  */
 void Irohad::initSimulator() {
   auto block_factory = std::make_unique<shared_model::proto::ProtoBlockFactory>(
-      std::make_unique<shared_model::validation::BlockVariantValidator>());
+      std::make_unique<
+          shared_model::validation::DefaultUnsignedBlockValidator>());
```

Comment:
Please make a comment here that "unsigned" also applies for the transaction signatures, besides the block signatures. It is not required to validate them here because they are validated in the ordering gate, where they are received from the ordering service.